app.js

```diff
@@ -0,0 +1,23 @@
+// const { application } = require('express');
+const express = require('express');
+const app = express();
+const { fetchAllTopics } = require('./controllers');
+
+
+app.use(express.json());
+
+app.get('/api/topics', fetchAllTopics)
+
+app.use((err, req, res, next) => {
```

Comment:
you've added this error handling but there's no test for this yet, at this stage all id expect to see in terms of error handling is a bad path test e.g `/api/notAPath`